### ISSUE TITLE
kcqrs-client: register and load requests using different version prefix

### ISSUE THREAD

**Issue by georgievJon** [clouway/kcqrs#29]
For registration of events is used path "/v1/aggregates" but for loading is used "/v2/aggregates". This is confusing and will be better if all path start with same version prefix.

**Comment by mgenov** [comment]
Yes, the loading endpoint is providing bulk support whether the registration is not. This change is required as if enforced same structure it will break up the existing clients.

**Comment by mlesikov** [comment]
Can we just rollback to  v1 - cause we  are still not in production and the path version staging is not required?

**Comment by StanislavaKaukova** [comment]
We can just add
 ```Spark.path("/v1/aggregates", {
      post("", SaveEventsHandler(eventStore, pubSubService), json())})
``` 
in eventstore-service and no need to change  version here

**Comment by mgenov** [comment]
We cannot rollback to v1 as Saving is not changed. 

e.g the endpoint POST /v1/aggregates is not changed and is kept as it is 

The GET /v2/aggregates is the newly introduced endpoint which should be added in implementation of the eventstore which is not part of the library.

**Comment by StanislavaKaukova** [comment]
yes, i know, but in eventstore-service we have only services for v2(now evenstore-service is not synchronized with ksqrs ) 
```
("/v2/aggregates", {
      post("", Save())
      get("", BulkGet())
      get("/:id",GetById())
    })
``` 

and i think to change post command in eventstore-service to
 ``` 
("/v1/aggregates", {
      post("", Save()})
```
instead to made changes in kcqrs lib

**Comment by mgenov** [comment]
I explicitly don't mention name of the services as they are internal to the company and shouldn't be used in an external communication. 

I think that the eventstore should have 2 GET endpoints: 

GET /v1/aggregates 
and 
GET /v2/aggregates 

After all clients are migrated to use the /v2 endpoint then the /v1/ could be deleted from the source tree. This will let both v1 and v2 to be supported by a single service.

**Comment by StanislavaKaukova** [comment]
Sorry, it will not happen again.

If I understand correctly in eventstore i will have 
get ```v1/aggregates(bulk)``` | ```v1/aggregates(by id) (i think this is no needed, because 
all now use v2 for get or i or i miss something)

get ```v2/aggregates(bulk)``` | ```v2/aggregates(by id)

post ```v1/aggregates```

**Comment by mgenov** [comment]
v1 endpoint is not going to support bulk gets. It should return the same response as it was in the past. The v2 on the other hand should support bulk operations.

**Comment by mgenov** [comment]
Post to /v1/aggregates should preserve it's signature.
